internal/runners/secrets/secrets.go

```diff
@@ -83,7 +83,7 @@ func (o *listOutput) MarshalStructured(format output.Format) interface{} {
 			out.HasValue = true
 		}
 
-		output = append(output, out)
+		output[i] = out
```

Comment:
Curious; what is this fixing?

Reply:
If there are no secrets, the array will be empty, and the JSON output will be `null` instead of `[]`. Not sure exactly why. I'm using the `[i]` notation here to avoid multiple calls to `append()` because we already know the length.

Reply:
Cool, thanks for the explanation!